yarn-project/end-to-end/bootstrap.sh

```diff
@@ -162,6 +162,9 @@ function generate_example_app_ivc_inputs {
     scripts/run_test.sh simple e2e_amm
     scripts/run_test.sh simple e2e_nft
     scripts/run_test.sh simple e2e_blacklist_token_contract/transfer_private
+    scripts/run-test.sh simple e2e_client_flows/deployments
+    scripts/run_test.sh simple e2e_client_flows/bridging
+    scripts/run_test.sh simple e2e_client_flows/transfers
```

Comment:
Fairly certain we don't want to run these along with the other e2e, we're not testing app or client behavior in them, we only care about the measurements.

Reply:
This is the bench command, but nonetheless I moved it